<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MangaReaderWidget</class>
 <widget class="QWidget" name="MangaReaderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="MangaImageWidget" name="mangaImageWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="readerFrontLightBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <property name="spacing">
          <number>10</number>
         </property>
         <item row="1" column="0">
          <widget class="QToolButton" name="toolButtonLessLight">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/images/icons/darker.png</normaloff>:/images/icons/darker.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Time</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="toolButtonMoreLight">
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/images/icons/lighter.png</normaloff>:/images/icons/lighter.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="toolButtonLessComfLight">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/images/icons/moon.png</normaloff>:/images/icons/moon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="horizontalSliderLight">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="toolButtonMoreComfLight">
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/images/icons/sun.png</normaloff>:/images/icons/sun.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="horizontalSliderComfLight">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>500</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="BatteryIcon" name="batteryIcon">
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>524</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="readerNavigationBar">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QLabel" name="labelReaderChapter">
             <property name="text">
              <string>Chapter</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonReaderGoto">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Goto...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelReaderPage">
             <property name="text">
              <string>Page</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="frameButtons">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButtonReaderHome">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Home</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="baseSize">
               <size>
                <width>1</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::VLine</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonReaderFavorites">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Favorites</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="baseSize">
               <size>
                <width>1</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::VLine</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonReaderBack">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Back</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BatteryIcon</class>
   <extends>QLabel</extends>
   <header>batteryicon.h</header>
  </customwidget>
  <customwidget>
   <class>MangaImageContainer</class>
   <extends>QFrame</extends>
   <header>mangaimagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
